exonum/src/storage/db.rs

```diff
@@ -354,7 +359,8 @@ impl Snapshot for Fork {
 }
 
 impl Fork {
-    /// Creates a new checkpoint.
+    /// Creates a new checkpoint. In Exonum checkpoints are created before
```

Comment:
Keep in mind, that it's better to separate the first short string of description from the more detailed part with one empty line.